radify/PR.js
# npm and jspm package updates

### Description
- Packages updates.
### Test
1. Clone a clean PR.js repo
2. Checkout branch `jspm-package-updates`
3. Follow [the installation instructions](https://github.com/radify/PR.js#get-going)
4. After running `gulp serve`, 
   - open [`http://localhost:3003`](http://localhost:3003)
   - **Assert that** the app loads properly in the browser
5. In the search box, enter "radify"
   - **Assert that** Radify's public repos display in the left hand column
6. Click on `angular-model`
   - **Assert that** one or more PRs load.
7. Click on `#22Revert should remove new properties`
   - **Assert that** you see Wolan
   - **Assert that** you see the complete PR message.
   - **Assert that** you see a mix of red and green test results
8. Click on `View`
   - **Assert that** it takes you to [the PR request on Github](https://github.com/radify/angular-model/pull/22)
